Simbody/include/simbody/internal/Force_Custom.h

```diff
@@ -247,6 +247,13 @@ class SimTK_SIMBODY_EXPORT Force::Custom::Implementation {
     virtual bool dependsOnlyOnPositions() const {
         return false;
     }
+    /**
+     * Called during the realizeSubsystemDynamicsImpl method in GenralForceSubsystemRep, used to determine whether a custom force
+     * should be parallelized. By default, this method returns false and can be overwritten by the force subclass.
+     */
+    virtual bool isParallelByDefault() const {
```

Comment:
This method is trying to say too much. It should just express something local that the Force-author can provide, which is whether the Force *should* be done in parallel. The author can't know what will actually happen. I would call it something like `isComputationallyExpensive()`. Then the comment can say that designating the force as expensive is a hint to Simbody that it would be worthwhile to parallelize it if possible.

That would make it like the previous method `dependsOnlyOnPositions()` which is something the author knows and is treated as a hint that certain optimizations can be performed.